src/Application/Cli/ClearCommand.php

```diff
@@ -29,13 +29,19 @@ public function __construct(Filesystem $filesystem)
         parent::__construct();
     }
 
+    /**
+     * {@inheritdoc}
+     */
```

Comment:
Could you please revert these? (it's adding unnecessary noise)

Reply:
Sure. Removing it everywhere instead then.